src/graphic/Fast3D/gfx_dxgi.h

```diff
@@ -20,4 +20,143 @@ void ThrowIfFailed(HRESULT res, HWND h_wnd, const char* message);
 
 extern "C" struct GfxWindowManagerAPI gfx_dxgi_api;
 
+#include "SDL_scancode.h"
+const SDL_Scancode windows_scancode_table[] = {
+    SDL_SCANCODE_UNKNOWN,
+    SDL_SCANCODE_ESCAPE,
+    SDL_SCANCODE_1,
+    SDL_SCANCODE_2,
+    SDL_SCANCODE_3,
+    SDL_SCANCODE_4,
+    SDL_SCANCODE_5,
+    SDL_SCANCODE_6, /* 0 */
```

Comment:
What are the commented numbers?

Reply:
The commented numbers are from copying over `gfx_sdl2`'s table, and were probably there to assist with enumerating. a similar system is in `scancodes_windows.h`.